tests/test_filters.py

```diff
@@ -0,0 +1,17 @@
+from django.test import TestCase
+
+from base.tests.factories.person import PersonFactory
+
+
+class AdmissionDoctorateFilterTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.doctorate_applicant = PersonFactory()
+
+    def test_filter_autocomplete_returns_existing_user(self):
+        self.client.get(
+            "admissions:person_autocomplete",
+            data={
+                "q": self.doctorate_applicant.email,
```

Comment:
```suggestion
        cls.doctorate_applicant = PersonFactory('foo@bar.example.org')

    def test_filter_autocomplete_returns_existing_user(self):
        self.client.get(
            "admissions:person_autocomplete",
            data={
                "q": 'bar',
```

et vérifier les données que tu reçois

Reply:
J'ai mis à jour le test pour vérifier que les données reçues sont formatées comme attendues.